src/plans/README.md

```diff
@@ -42,4 +42,22 @@ See the comments inside all the plans for more information.
    the ground, which creates joint over-torquing warnings and errors.
 
 6. Continuous: non-terminating plan that performs continuous operations, useful
-   as a stress/load test.
\ No newline at end of file
+   as a stress/load test.
+
+Testing plans
+-------------
+
+### ReferenceMission2 ###
+
+- How to run:
+	- Launch OceanWATERS ([Launch Tutorial](https://github.com/nasa/ow_simulator/wiki/Tutorials))
+	- Source your OceanWATERS workspace
+	- Run the autonomy node with this plan as the argument:
+	  `roslaunch ow_autonomy autonomy_node.launch plan:=ReferenceMission2.plx`
+
```

Comment:
This part can be omitted, because it's covered in ow_autonomy/README.md (which is the file displayed on the ow_autonomy GitHub page).

Reply:
Makes sense, I figured that was a bit redundant. I will update it.